node/impl/full/eth_utils.go

```diff
@@ -651,7 +651,7 @@ func newEthTxFromMessageLookup(ctx context.Context, msgLookup *api.MsgLookup, tx
 	return newEthTx(ctx, cs, st, parentTs.Height(), parentTsCid, msgLookup.Message, txIdx)
 }
 
-func newEthTx(ctx context.Context, cs *store.ChainStore, st *state.StateTree, blockHeight abi.ChainEpoch, parentTsCid cid.Cid, msgCid cid.Cid, txIdx int) (ethtypes.EthTx, error) {
+func newEthTx(ctx context.Context, cs *store.ChainStore, st *state.StateTree, blockHeight abi.ChainEpoch, msgTsCid cid.Cid, msgCid cid.Cid, txIdx int) (ethtypes.EthTx, error) {
 	smsg, err := getSignedMessage(ctx, cs, msgCid)
```

Comment:
@virajbhartiya @rvagg 

`msgTsCid` is the cid of the block that "included" the message, not the one that "executed" it. 
`parentTsCid` was a misnomer for the name and so fixed it.